eventlet/support/greendns.py

```diff
@@ -180,10 +180,11 @@ def _readlines(self):
         """
         lines = []
         try:
-            with open(self.fname, 'rU') as fp:
+            with open(self.fname, 'r') as fp:
```

Comment:
That's not only dictatory rule, but also a practical reason for Python2 tests fail, see Travis build link below.